### PR TITLE
Admins script

### PR DESCRIPTION
**Pull request by groovecoder**

### CODE REVIEW

File: admins.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#!/usr/bin/env python
+import urllib
+
+import requests
+
+from client import get_token, get_github3_client
+
+if __name__ == '__main__':
+    # only using this for members_urlt
+    gh = get_github3_client()
```
Comment (fwenzel) [inline admins.py]: Doesn't this send a login request? Do we need to fully instantiate a gh client instance just to get to the URL template?

File: admins.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    # the rest is raw requests.get
+    headers = {
+        'Accept': 'application/vnd.github.moondragon+json',
+        'Authorization': 'token %s' % get_token()
+    }
+    params = {'filter': '2fa_disabled',
```
Comment (fwenzel) [inline admins.py]: I'm surprised the library itself does not let you feed it a filter

**Comment by fwenzel** [comment]
Sweet PR, thanks. I'm mostly curious about the things I asked here, otherwise this is good to merge!

File: admins.py (hunk 1 above)
Comment (groovecoder) [inline admins.py]: It does. I'm 50/50 on it - would be nice not to repeat the members URL template in the library and in the app code, but ... :neutral_face:

File: admins.py (hunk 2 above)
Comment (groovecoder) [inline admins.py]: Yeah, it seems to sacrifice versatility for more wrapped "pythonic" usage.

**Comment by groovecoder** [comment]
Updated to remove the unnecessary gh login requirement.

**Comment by fwenzel** [comment]
Awesome thanks!

**Pull request merged**